assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/TicketPriceField/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './style.scss';
+
+interface TicketPriceFieldProps {
+	after: string;
+	before: string;
+	field: React.ReactNode;
+}
+
+const TicketPriceField: React.FC<TicketPriceFieldProps> = ({ after, before, field }) => {
```

Comment:
I've created this component to consolidate `ee-ticket-price-field` and it's `after` and `before`, because it was scattered in 2 place and as a result we did had great control.

Reply:
ya great idea... i have some requests though (see comment on `TicketPrice.tsx`)